src/index.html

```diff
@@ -1,16 +1,51 @@
 <!doctype html>
-<html lang="en">
+  <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      data-qa="big"
+      method="GET"
+    >
+      <label class="search-big shadow" for="search-big">
+        <img src="images/Search.svg"
+          alt="search-icon-big"
+          class="search-icon"
+        >
+        <input
+          type="search"
+          data-qa="keypress"
+          class="search-inp"
```

Comment:
```suggestion
          class="search-input"
```
Чому 'inp' а не 'input'? Виправ це